documentation/asciidoc/accessories/display/legacy.adoc

```diff
@@ -1,6 +1,30 @@
 == Legacy Support
 
-NOTE: These instructions are for the original Raspberry Pi 1 Model A and B boards only. You can identify an original board as it is the only model with a 26-pin GPIO header, all other models have the now-standard 40-pin connector.
+[NOTE]
+====
+Older Raspberry Pi hardware (pre-Raspberry Pi 4), and older versions of Raspberry Pi OS (pre-Bookworm), with X11 can install an on-screen keyboard and emulate right-clicks on the touchscreen. 
+
+Run the following in a terminal to install the on-screen keyboard:
+----
+sudo apt install matchbox-keyboard
+----
+
+Add the following to `/etc/X11/xorg.conf` enable right-click emulation:
+
+----
+Section "InputClass"
+   Identifier "calibration"
+   Driver "evdev"
+   MatchProduct "FT5406 memory based driver"
+
+   Option "EmulateThirdButton" "1"
+   Option "EmulateThirdButtonTimeout" "750"
+   Option "EmulateThirdButtonMoveThreshold" "30"
+EndSection
+----
+====
+
+NOTE: The following instructions are for the original Raspberry Pi 1 Model A and B boards only. You can identify an original board as it is the only model with a 26-pin GPIO header, all other models have the now-standard 40-pin connector.
```

Comment:
Avoid "following". Also NOTE blocks are stand alone pieces of text that should not be used to modify the narrative. They're call out blocks, asides, and footnotes.